FIO/local-volumes/dbench-job-generator-local-volume-replica.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+set -euo pipefail
+
+#
+# The following script provisions a local volume with one replica and runs fio tests
```

Comment:
We should rephrase slightly here. Technically, we're provisioning a volume with a replica and then launching a pod on the same node as the master - this defines it as 'local'. There is no such thing as a 'local volume'.